src/gui/sdl_gui.cpp

```diff
@@ -1714,7 +1714,7 @@ void GFX_InitSdl()
 	set_sdl_hints();
 
 	// Initialise SDL (timer is needed for title bar animations)
-	if (SDL_InitSubSystem(SDL_INIT_AUDIO | SDL_INIT_VIDEO | SDL_INIT_TIMER) < 0) {
+	if (SDL_InitSubSystem(SDL_INIT_VIDEO | SDL_INIT_TIMER) < 0) {
 		E_Exit("SDL: Can't init SDL %s", SDL_GetError());
```

Comment:
Probably would make sense to be more explicit in the log message now: "Failed to init SDL video and timer: %s" perhaps?